src/checkpackage.wsgi

```diff
@@ -2,11 +2,9 @@
 from webob import Request
 
 from retrace.retrace import (INPUT_ARCH_PARSER,
-                             INPUT_PACKAGE_PARSER,
```

Comment:
`INPUT_PACKAGE_PARSER` is used below on line 20. Also, you moved these parsers into `util`.